src/org/freedesktop/gstreamer/meta/GstMeta.java

```diff
@@ -0,0 +1,22 @@
+package org.freedesktop.gstreamer.meta;
+
+import java.util.stream.Stream;
+import org.freedesktop.gstreamer.glib.NativeObject;
+import static org.freedesktop.gstreamer.glib.Natives.registration;
+
+/**
+ * @author Jokertwo
+ */
+public class GstMeta implements NativeObject.TypeProvider {
```

Comment:
Should be called Meta. Should specific Meta extend from this?  What is the Meta hierarchy?  Java class hierarchy should generally match with GType hierarchy.

The `Type.Provider` implementation should probably be nested, like `Message.Types`